suite-native/module-accounts-import/src/components/AccountImportOverview.tsx

```diff
@@ -64,7 +51,10 @@ export const AccountImportOverview = ({ accountInfo, networkSymbol }: AssetsOver
             </Box>
             <Box marginBottom="large">
                 <Text variant="titleLarge" color="gray1000">
-                    {FiatAmountFormatter.format(fiatAmount ?? 0)}
+                    {CryptoToFiatAmountFormatter.format({
```

Comment:
Prefer component version of formatters.

Reply:
https://github.com/trezor/trezor-suite/pull/7401/commits/1cc34593cd42d928be9c517ad6ad4da0f00dc9a3